client/signup/jetpack-connect/plans.jsx

```diff
@@ -69,6 +70,10 @@ class Plans extends Component {
 	}
 
 	componentDidUpdate() {
+		if ( this.props.isAutomatedTransfer && ! this.redirecting ) {
+			this.redirect( CALYPSO_REDIRECTION_PAGE );
```

Comment:
should we redirect to wp-admin instead? That's what happens if you skip the plans page by selecting the "free" plan.